Modules/timemodule.c

```diff
@@ -1373,6 +1398,16 @@ _PyTime_GetThreadTimeWithInfo(_PyTime_t *tp, _Py_clock_info_t *info)
 
 #elif defined(HAVE_CLOCK_GETTIME) && defined(CLOCK_PROCESS_CPUTIME_ID)
 #define HAVE_THREAD_TIME
+
+#ifdef __APPLE__
```

Comment:
Do we also want to check for `__has_attribute(availability)` here?

Reply:
I guess so. The API is new in 10.13 and the version of Xcode with the 10.13 SDK always supports this attribute, but someone might compile with GCC.